shared/team-building/container.tsx

```diff
@@ -170,7 +167,7 @@ const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
   return {
     ...contactProps,
     disabledRoles,
-    recommendations: deriveSearchResults(
+    recommendations: deriveRecommendation(
```

Comment:
@cjb this called the memoized function 2 times in a row w/ different values, effectively killing the memo